GateWatcher/aioniq/ingest/parser.yml

```
@@ -202,3 +203,14 @@ stages:
           gatewatcher.targeted_countries: "{{json_load.message.targeted_countries}}"
           gatewatcher.targeted_platforms: "{{json_load.message.targeted_platforms}}"
           gatewatcher.targeted_organizations: "{{json_load.message.targeted_organizations}}"
+  tls:
+    actions:
+      - set:
+         tls.server.issuer: "{{json_load.message.tls.issuerdn}}"
+         tls.server.not_before: "{{json_load.message.tls.notbefore}}"
+         tls.server.certificate_chain: "{{json_load.message.tls.chain}}"
+         tls.server.subject: "{{json_load.message.tls.subject}}"
+         gatewatcher.tls.sni: "{{json_load.message.tls.sni}}"
+         gatewatcher.tls.fingerprint: "{{json_load.message.tls.fingerprint}}"
```

According to the comment above:
```suggestion
         gatewatcher.tls.sni: "{{json_load.message.tls_sni}}"
         gatewatcher.tls.fingerprint: "{{json_load.message.tls_fingerprint}}"
```